Undefined name: 'cpu_soft_nms'  in utils/utils.py

[flake8](http://flake8.pycqa.org) testing of https://github.com/Zhongdao/Towards-Realtime-MOT on Python 3.8.0

$ __flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics__
```
./utils/utils.py:421:12: F821 undefined name 'cpu_soft_nms'
    keep = cpu_soft_nms(np.ascontiguousarray(dets, dtype=np.float32),
           ^
1     F821 undefined name 'cpu_soft_nms'
1
```
__E901,E999,F821,F822,F823__ are the "_showstopper_" [flake8](http://flake8.pycqa.org) issues that can halt the runtime with a SyntaxError, NameError, etc. These 5 are different from most other flake8 issues which are merely "style violations" -- useful for readability but they do not effect runtime safety.
* F821: undefined name `name`
* F822: undefined name `name` in `__all__`
* F823: local variable name referenced before assignment
* E901: SyntaxError or IndentationError
* E999: SyntaxError -- failed to compile a file into an Abstract Syntax Tree

--- a/utils/utils.py
+++ b/utils/utils.py
@@ -414,13 +414,6 @@ def pooling_nms(heatmap, kernel=1):
     keep = (hmax == heatmap).float()
     return keep * heatmap
 
-def soft_nms(dets, sigma=0.5, Nt=0.3, threshold=0.05, method=1):
-    keep = cpu_soft_nms(np.ascontiguousarray(dets, dtype=np.float32),
-            np.float32(sigma), np.float32(Nt),
-            np.float32(threshold),
-            np.uint8(method))
-    return keep
-
 def non_max_suppression(prediction, conf_thres=0.5, nms_thres=0.4, method='standard'):
     """
     Removes detections with lower object confidence score than 'conf_thres'
@@ -431,7 +424,7 @@ def non_max_suppression(prediction, conf_thres=0.5, nms_thres=0.4, method='stand
         prediction,
         conf_thres,
         nms_thres,
-        method = 'standard', 'fast', 'soft_linear' or 'soft_gaussian'
+        method = 'standard' or 'fast'
     """
 
     output = [None for _ in range(len(prediction))]
@@ -457,12 +450,6 @@ def non_max_suppression(prediction, conf_thres=0.5, nms_thres=0.4, method='stand
         # Non-maximum suppression
         if method == 'standard':
             nms_indices = nms(pred[:, :4], pred[:, 4], nms_thres)
-        elif method == 'soft_linear':
-            dets = pred[:, :5].clone().contiguous().data.cpu().numpy()
-            nms_indices = soft_nms(dets, Nt=nms_thres, method=0)
-        elif method == 'soft_gaussian':
-            dets = pred[:, :5].clone().contiguous().data.cpu().numpy()
-            nms_indices = soft_nms(dets, Nt=nms_thres, method=1)
         elif method == 'fast':
             nms_indices = fast_nms(pred[:, :4], pred[:, 4], iou_thres=nms_thres, conf_thres=conf_thres)
         else:

